nodejs/week1/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,13 @@
+
+const express = require("express");
+const router = express.Router();
+const meals = require("../data/meals.json")
+
+router.get("/", (req, res)=>{
+    cheapMeals = meals.filter(cheapMeal => cheapMeal.price < 65)   
```

Comment:
This looks great!

There's one small addition that needs to be done. The reviews for a given meal need to be added to that meal and provided back in the response.